changelog/1.19.0.md

```diff
@@ -5,43 +5,41 @@ description: "Released on 05/19/2021"
 
 ### Breaking changes ❗
 
-- infra: Workspace assets created by Coder previously located at `/opt/coder`
-  such as code-server and coder-cli are now located in `/var/tmp/coder`
+- infra: Workspace assets created by Coder previously located at `/opt/coder`,
+  such as code-server and coder-cli, have been moved to `/var/tmp/coder`
 
 ### Features ✨
 
-- web: Resource quotas have been introduced to Organizations. Resource Quotas
-  define maximum resource utilization applied to each member within the
-  Organization they're applied to. Modify them **Manage > Organizations > Edit
-  Organization**.
-- web: Add site and organization workspace as code templates
-- web: An administrative control over Dev URL access levels was added.
-- web: By default, the CVM option for creating custom environments is now
-  checked
-- web: Improved display and support for errors returned by an IdP during sign in
-- infra: Change permissions of `/home/coder/` from _755_ to _744_
-- infra: Validate workspace builds when setting policy templates
-- infra: Add client certificate authentication support
+- web: Introduced resource quotes to organizations. Resource quotas define the
+  maximum resource utilization allowable for each member within an organization
+  for which such quotas are enabled. Modify them at **Manage** >
+  **Organizations** > **Edit Organization**.
+- web: Added site and organization workspace as code templates.
+- web: Added administrative control over dev URL access levels.
+- web: The CVM option when creating a custom environment is now selected by default.
+- web: Improved display and support for errors returned by an IdP during sign-in
+- infra: Changed permissions of `/home/coder/` from _755_ to _744_
```

Comment:
I'll find out - good call out

Reply:
(I'd assume only in workspace containers?)

Reply:
I think the change that @mterhar was to change /home/coder itself from 700 to 755, so that when we run with anyuid on OpenShift, the random uid can execute our binary. This was only in the control plane containers.

Reply:
It actually affects both the coder-service and dashboard containers